drogue-tls-sys/include/string.h

```diff
@@ -0,0 +1,12 @@
+#define NULL ((void*)0)
```

Comment:
Are the headers not included by clang in fedora?

Reply:
It doesn't look like. To be honest, I didn't dig too much into that.

Compilation complained but the files missing. And the noted during compilation that these are "built in" functions. I guess that is why linking works later on.

It also complains of the definition in the file does not match the internal definition.

Maybe there is some alternative "built-in" include directory that needs to be added into the include path, I can check that.

Reply:
Let's drag in @jcrossley3 maybe, since he also Fedoras...